packages/langium/src/lsp/language-server.ts

```diff
@@ -168,6 +168,14 @@ export function addDocumentHighlightsHandler(connection: Connection, services: L
     });
 }
 
+export function addHoverHandler(connection: Connection, services: LangiumServices): void {
+    const hoverProvider = services.lsp.HoverProvider;
+    connection.onHover(params => {
+        const document = paramsDocument(params, services);
+        return document && hoverProvider.getHoverContent(document, params.position);
```

Comment:
Please pass the whole `params` so we don't need to change our API in case LSP adds more properties.